lib/tether/tether.go

```diff
@@ -251,20 +252,32 @@ func (t *tether) setNetworks() error {
 }
 
 func (t *tether) setMounts() error {
+	// provides a lookup from path to volume reference.
+	NameLookupMap := make(map[string]string, 0)
```

Comment:
@caglar10ur it looks like I should be in this area again soon to address another potential bug. I may make this change at that time in order to avoid additional small commits onto the feature branch.